src/Xrefcheck/Verify.hs

```diff
@@ -706,6 +743,7 @@ checkExternalResource Config{..} link
       , (405 ==)  -- method mismatch
       ]
 
+    interpretErrors :: Network.HTTP.Req.HttpException -> Either VerifyError ()
```

Comment:
I think that this same `HttpException` is already imported in this file, so there is no need to use its full name.